Website/htdocs/mpmanager/resources/assets/js/components/MiniGrid/Dashboard.vue

```diff
@@ -760,7 +761,7 @@
             getTransactionsOverview () {
                 axios.post('/api/mini-grids/' + this.miniGridId + '/transactions', {
                     startDate: this.highlighted.base.from,
-                    endDate: this.highlighted.base.to
+                    endDate: this.highlighted.compared.to
```

Comment:
what happens when compared.to < base.from?  Please do a re-check and use them according to the result